invenio_app_ils/documents/loaders/jsonschemas/document.py

```diff
@@ -244,7 +244,7 @@ class Meta:
     imprint = fields.Nested(ImprintSchema)
     internal_notes = fields.List(fields.Nested(InternalNoteSchema))
     keywords = fields.List(fields.Nested(KeywordSchema))
-    languages = fields.List(fields.Str())
+    languages = fields.List(fields.Str(), required=True)
```

Comment:
did you check how does this impact the migration ?

Reply:
Currently 152885/154029 (99%) documents have the language field (041__a). IMHO it's ok for the migration

Reply:
and what happens in case of the 1% that does not have the language? do we set up the default?

Reply:
For the record, after talking with the librarians, they will clean the data.